.changeset/shaggy-ladybugs-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-js': minor
+---
+
+Add error states in form-behavior
```

Comment:
I think the changelog message can be a bit more specific. Perhaps something like "Add handling of form submission error state in form behavior"?